doc/Requirements Specification/Parts/Non-Functional Requirements.lyx

```diff
@@ -918,12 +914,12 @@ gls{SoMoX}
 
 \end_inset
 
- has finished.
+ .
 \end_layout
 
 \begin_layout Standard
-When the users launches the analysis, he gets an window where he can change
- some settings for 
+When the user launches the analysis, they are presented a window in where
```

Comment:
> I don’t think you can say “in where”, can you?

You're right

> Would you consider using “the users”?

This is a perfect example where it sounds stupid to use the plural form. But I won't argue and use it.